### PR TITLE
Changed logout button to red

### PR DESCRIPTION
**Pull request by Nate-Gage**
changed logout button to red.

### CODE REVIEW

File: src/components/LoginUser.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ function LoginUser() {
                 </form>
                 {loginMsg && <p>Successfully logged in!</p>}
                 {userValue ?
-                    <button className="loginBtn btn btn-success"
+                    <button className="loginBtn btn btn-danger"
```
Comment (gagej) [inline src/components/LoginUser.js:53]: don't use the danger css, use the happy css

File: src/components/LoginUser.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ function LoginUser() {
                 </form>
                 {loginMsg && <p>Successfully logged in!</p>}
```
Comment (gagej) [inline src/components/LoginUser.js:51]: one more thing: don't use an exclamation point

Comment (Nate-Gage) [reply src/components/LoginUser.js:51]: Story required exclamation point

Comment (gagej) [reply src/components/LoginUser.js:51]: oh right, I forgot about that

File: src/components/LoginUser.js (hunk 1 above)
Comment (Nate-Gage) [reply src/components/LoginUser.js:53]: Sounds good

**Pull request merged**